From 0547915e9d5ffb471203fd3808e8e16f94a6fa83 Mon Sep 17 00:00:00 2001
From: Bryan O'Donoghue <bryan.odonoghue@linaro.org>
Date: Thu, 16 Nov 2023 16:56:36 +0000
Subject: [PATCH 123/173] johan_defconfig: Switch off CONFIG_LOCALVERSION_AUTO

My build scripts are built around a static name.
Drop LOCALVERSION_AUTO.

Signed-off-by: Bryan O'Donoghue <bryan.odonoghue@linaro.org>
---
 arch/arm64/configs/johan_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/johan_defconfig b/arch/arm64/configs/johan_defconfig
index d90712731762..dd1c99171f01 100644
--- a/arch/arm64/configs/johan_defconfig
+++ b/arch/arm64/configs/johan_defconfig
@@ -1,3 +1,4 @@
+# CONFIG_LOCALVERSION_AUTO is not set
 CONFIG_SYSVIPC=y
 CONFIG_POSIX_MQUEUE=y
 CONFIG_AUDIT=y
-- 
2.43.0

